modules/openapi-generator/src/main/resources/python/model_test.mustache

```diff
@@ -5,12 +5,17 @@
 from __future__ import absolute_import
 
 import unittest
+import datetime
+from io import StringIO
 
 {{#models}}
 {{#model}}
 import {{packageName}}
 from {{modelPackage}}.{{classFilename}} import {{classname}}  # noqa: E501
 from {{packageName}}.rest import ApiException
+{{#vars}}{{#required}}{{^isPrimitiveType}}{{#isModel}}{{^isEnum}}
+from test import Test{{dataType}}
```

Comment:
how about from test import *?
Then we can use TestCat or CatExample classes below?

Reply:
Can you resolve this conversation?